test/jobs/zendesk_job_test.rb

```diff
@@ -0,0 +1,61 @@
+require_relative '../test_helper'
+
+SingleCov.covered!
+
+describe 'Zendesk Job' do
+  hostname = 'https://hostname.com'
+  username = 'username@email.com'
+  password = '123456'
+  describe 'boostraps zendesk job' do
+    it 'bootstraps by getting environment variables' do
+      ENV['ZENDESK_HOSTNAME'] = hostname
+      ENV['ZENDESK_USERNAME'] = username
+      ENV['ZENDESK_PASSWORD'] = password
+      ZendeskJob.bootstrap()
+      assert ZendeskJob.hostname == hostname
+      assert ZendeskJob.username == username
+      assert ZendeskJob.password == password
+    end
+  end
+  
+  describe 'has zendesk integration' do
+    it 'should return true if zendesk variables are set' do
+      ZendeskJob.hostname = hostname
+      ZendeskJob.username = username
+      ZendeskJob.password = password
+      assert ZendeskJob.has_zendesk_integration?() == true
```

Comment:
no parens with no args

Reply:
booleans are a good case for using plain `assert`, but adding the second argument as a failure message

```rb
assert ZendeskJob.has_zendesk_integration?, 'must return true'
```

this way when you put together you test context, you get a nicer message like `has zendesk integration :: is true if zendesk variables are set :: must return true`